bpf/types.h

```diff
@@ -89,37 +89,30 @@ typedef struct flow_metrics_t {
     // https://chromium.googlesource.com/chromiumos/docs/+/master/constants/errnos.md
     u8 errno;
     u8 dscp;
+    struct dns_record_t {
+        u16 id;
+        u16 flags;
+        u64 latency;
+        u8 errno;
+    } dns_record;
```

Comment:
why we keep dns record here and not move to additional metrics ?

Reply:
yeah I mentioned that in the TODO here https://github.com/netobserv/netobserv-ebpf-agent/pull/470
I didn't do it at first because it's modified from the TC hook so it isn't required to move it ... but still it would probably be better to do it.

Reply:
addressed via https://github.com/netobserv/netobserv-ebpf-agent/pull/469/commits/01950d5b862e9ce494de90caf87a8846baa4b89d